test/performance/odp_packet_gen.c

```diff
@@ -183,11 +214,18 @@ static void print_usage(void)
 	       "  -o, --udp_src             UDP source port. Default: 10000\n"
 	       "  -p, --udp_dst             UDP destination port. Default: 20000\n"
 	       "  -P, --promisc_mode        Enable promiscuous mode.\n"
+	       "  -a, --latency             Calculate latency. Not available with packet\n"
+	       "                            references (see \"--no_pkt_refs\").\n"
```

Comment:
Perhaps it would be simpler to a user that enabling latency measurement would disable references automatically. For example: `Automatically disables packet references (see --no_pkt_refs).`

Reply:
V2.